src/components/BookList.spec.js

```diff
@@ -33,9 +34,92 @@ describe('Book list', () => {
   ]
 
   it('renders two listitems', () => {
-    render(<BookList books={books} />)
+    const mockOnHandleActiveReadingStatus = jest.fn()
+
+    render(
+      <BookList
+        books={books}
+        handleActiveReadingStatus={mockOnHandleActiveReadingStatus}
```

Comment:
`BookList` doesn't have the callback function as props. So this line doesn't do anything 🙂